processor/memorylimiterprocessor/config.go

```diff
@@ -1,68 +0,0 @@
-// Copyright The OpenTelemetry Authors
-// SPDX-License-Identifier: Apache-2.0
-
-// Package memorylimiterprocessor provides a processor for OpenTelemetry Service pipeline
-// that refuses data on the pipeline according to the current state of memory usage.
-package memorylimiterprocessor // import "go.opentelemetry.io/collector/processor/memorylimiterprocessor"
-
-import (
-	"errors"
-	"time"
-
-	"go.opentelemetry.io/collector/component"
-)
-
-var (
-	errCheckIntervalOutOfRange        = errors.New("'check_interval' must be greater than zero")
-	errLimitOutOfRange                = errors.New("'limit_mib' or 'limit_percentage' must be greater than zero")
-	errSpikeLimitOutOfRange           = errors.New("'spike_limit_mib' must be smaller than 'limit_mib'")
-	errSpikeLimitPercentageOutOfRange = errors.New("'spike_limit_percentage' must be smaller than 'limit_percentage'")
-	errLimitPercentageOutOfRange      = errors.New(
-		"'limit_percentage' and 'spike_limit_percentage' must be greater than zero and less than or equal to hundred")
-)
-
-// Config defines configuration for memory memoryLimiter processor.
-type Config struct {
```

Comment:
Every exported API must be preserved. Otherwise, it's a breaking change. Please keep an alias here. `var Config = memorylimiter.Config`

Reply:
ok all back in place. didn't realize

Reply:
You don't need to restore it as is. Let's avoid code duplication. Just replace the declaration with the alias `var Config = memorylimiter.Config`